backend/src/test/java/com/google/sps/FindMatchQueryTest.java

```diff
@@ -99,53 +110,75 @@ public void compatibleTime() {
   @Test
   public void compatibleDuration() {
     // Two participants who are compatible in duration but NOT available time
-    Participant participantA = new Participant(ID_DEFAULT, PERSON_A, TIME_0250PM, TIMEZONE_DEFAULT, DURATION_45_MINUTES, TIMESTAMP_DEFAULT);
-    Participant participantB = new Participant(ID_DEFAULT, PERSON_B, TIME_0400PM, TIMEZONE_DEFAULT, DURATION_60_MINUTES, TIMESTAMP_DEFAULT);
-    
+    Participant participantA =
+        new Participant(
+            ID_DEFAULT, PERSON_A, TIME_0200PM, TIME_0250PM, DURATION_45_MINUTES, TIMESTAMP_DEFAULT);
+    Participant participantB =
+        new Participant(
+            ID_DEFAULT, PERSON_B, TIME_0200PM, TIME_0400PM, DURATION_60_MINUTES, TIMESTAMP_DEFAULT);
+
     Match match = query.findMatch(Arrays.asList(participantA), participantB);
-    
+
     assertThat(match).isNull();
   }
 
   @Test
   public void threeParticipants13() {
     // Three participants, 1st and 2nd aren't compatible, but 1st and 3rd are
```

Comment:
nit: the tests have participant A, B, and C, but the comments and test names are in terms of 1, 2, and 3 (this is the same comment i have for the following tests)

Reply:
Ooh good catch! Thanks, done.